clients/drcachesim/tests/scheduler_unit_tests.cpp

```diff
@@ -160,6 +166,101 @@ verify_scheduler_stats(scheduler_t::stream_t *stream, int64_t switch_input_to_in
            migrations);
 }
 
+// Returns a string with one char per input.
+// Assumes the input threads are all tid_base plus an offset < 26.
+// When send_time=true, typically time_units_per_us should be set to 1 to then have
+// instruction count for all timing measures.
```

Comment:
That last sentence isn't very clear.  Looking at the code, it appears it assumes IPC=1, which is a bit stronger than the word "typically" implies.  Also, I'm not sure how to read "have instruction count for all timing measures".

Reply:
Rewrote as:

// When send_time=true, the record count is passed to the scheduler as the current
// time, to avoid relying on wall-clock time.  For this use case of send_time=true,
// typically time_units_per_us should be set to 1 to avoid any scaling of the record
// count for simpler small tests.